### PR TITLE
Mount hollow-node logs to parent node hostpath

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: test/kubemark/resources/hollow-node_template.json [hunk 1]
```diff
@@ -45,20 +51,29 @@
...
 					"command": [
-						"./kubemark.sh"
-					],
-					"args": [
-						"--v=3",
-						"--morph=kubelet",
-						"$(CONTENT_TYPE)"
+						"/bin/sh",
+						"-c",
+						"./kubemark.sh --morph=kubelet $(CONTENT_TYPE) --v=2 1>>/var/logs/kubelet_$(MY_POD_NAME).log 2>&1"
```
Comment (gmarek) [inline test/kubemark/resources/hollow-node_template.json]: I just wish logDir was exported variable...

Comment (wojtek-t) [inline test/kubemark/resources/hollow-node_template.json]: We can change it at some point.

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please, issue: #26135

File: test/kubemark/resources/hollow-node_template.json (hunk 1 above)
Comment (gmarek) [inline test/kubemark/resources/hollow-node_template.json]: golang guys would need to...

**Comment by fejta** [comment]
@k8s-bot node e2e test this issue: #27296

**Comment by fejta** [comment]
@k8s-bot unit test this issue: #28076

**Pull request merged**